src/modules/Auth/Login/LoginForm.tsx

```diff
@@ -0,0 +1,118 @@
+import { useForm, Controller } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useTranslation } from "react-i18next";
+import { Checkbox } from "antd";
+
+import { CommonButton, PasswordField, TextField } from "components";
+import { ILogin } from "types/auth.model";
+import { useAppDispatch } from "hooks/useRedux";
+import {
+  showForgotPswFormModal,
+  showSignUpFormModal
+} from "redux/features/auth.slice";
+import { initialLoginFormValues, loginFormSchema } from "./Login.constants";
+import {
+  ForgotPswText,
+  LoginTitle,
+  NotHaveAccount,
+  PasswordWrapper,
+  RememberPswText,
+  RememberPswWrapper
+} from "./LoginForm.styled";
+
+const LoginForm = () => {
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+
+  const handleNavigateToAuthModule = (mode: "sign-up" | "forgot-psw") => {
+    mode === "sign-up" && dispatch(showSignUpFormModal());
```

Comment:
use if( mode === "login" ) instead of double ampersand (&& )

Reply:
To easy to understand or purpose ?